decimal.go

```diff
@@ -108,15 +125,51 @@ func NewFromInt(value int64) Decimal {
 //
 // Example:
 //
-//     NewFromInt(123).String() // output: "123"
-//     NewFromInt(-10).String() // output: "-10"
+//	NewFromInt(123).String() // output: "123"
+//	NewFromInt(-10).String() // output: "-10"
 func NewFromInt32(value int32) Decimal {
 	return Decimal{
 		value: big.NewInt(int64(value)),
 		exp:   0,
 	}
 }
 
+// MulInt calculates d * n value.
+func MulInt(value Decimal, n int) Decimal {
+	d := NewFromInt(int64(n))
+	return value.Mul(d)
+}
```

Comment:
This could be a receiver function I think

```suggestion
func (d Decimal) MulInt(n int) Decimal {
	nd := NewFromInt(int64(n))
	return d.Mul(nd)
}
```